tests/ScopesTest.php

```diff
@@ -21,6 +21,25 @@ public function test_translated_in_scope_works_with_default_locale()
         $this->assertSame('Griechenland', $translatedCountries->first()->name);
     }
 
+    public function test_not_translated_in_scope_returns_only_not_translated_records_for_this_locale()
+    {
+        $notTranslatedCountries = Country::notTranslatedIn('en')->get();
+
```

Comment:
This is really great! Can you please assert here that `count(notTranslatedCountries)` is positive?